src/xbot/common/logging/PeriodicDeltaObserver.java

```diff
@@ -0,0 +1,30 @@
+package xbot.common.logging;
+
+import org.apache.log4j.Logger;
+
+public class PeriodicDeltaObserver {
+
+    protected static Logger log = Logger.getLogger(PeriodicDeltaObserver.class);
+    
+    private double checkPowerThreshold;
+    
+    private double oldPower = 0;
```

Comment:
consider going with `value` instead of `power` since this is more generic than motor powers?